pylib/Makefile

```diff
@@ -1,5 +1,21 @@
 SHELL := /bin/bash
 FIND := $(if $(wildcard /bin/find),/bin/find,/usr/bin/find)
+MYPY_IGNORED_IMPORTS :=
+
+ifndef OS
+	OS := unknown
+endif
+
+ifndef UNAME
+	UNAME := unknown
+endif
+
+# https://anki.tenderapp.com/discussions/beta-testing/1860-error-unused-type-ignore-comment
+ifneq (${OS},Windows_NT)
+	ifneq (${UNAME},Darwin)
```

Comment:
I don't think you need to exclude macOS, the tests work fine here.

Reply:
Done:
```
ifneq (${OS},Windows_NT)
	MYPY_ARGS := --warn-unused-ignores
endif
```